kinesis-video-gst-demo/kinesis_video_gstreamer_audio_video_sample_app.cpp

```diff
@@ -56,7 +56,7 @@ LOGGER_TAG("com.amazonaws.kinesis.video.gstreamer");
 #define APP_SINK_BASE_NAME "appsink"
 #define DEFAULT_BUFFER_SIZE (1 * 1024 * 1024)
 #define DEFAULT_STORAGE_SIZE (128 * 1024 * 1024)
-#define DEFAULT_CREDENTIAL_ROTATION_SECONDS 2400
+#define DEFAULT_CREDENTIAL_ROTATION_SECONDS 3600
```

Comment:
How does changing creds rotation from 40 minutes to 60 minutes help?

Reply:
This is not the real root cause but rather arbitrary value. Most of the session tokens issued have an hour life span so it's better to mimmic it.